### ISSUE TITLE
Does GET work?

### ISSUE THREAD

**Issue by alapolloni** [tableau/tabcmd#148]
Does GET currently work with the python version?  Some of the other commands like LOGIN and LISTSITES work for me but GET doesn't.  GET is working for me using the old executable version.  

I get this for python GET:
INFO:  Connecting to the server...
INFO:  Succeeded
INFO:  Unexpected response from the server: 

(can I get any further error logging?)

Tableau Server Command Line Utility v2.0.5

**Comment by bcantoni** [comment]
@alapolloni could you clarify with an example what type of Get are you trying? We have a few bugs in process of being fixed and it would be helpful to know if your example is one of them. If you could show your tabcmd command line, that would be helpful. (Or a sanitized version of it if any names are sensitive.)

**Comment by alapolloni** [comment]
(apologies, just back from holidays)

Hi @bcantoni , I tried many iterations of the examples from https://tableau.github.io/tabcmd/docs/tabcmd_cmd.html#examples-3

here's one example:

```
(base) ala@XXXXXXXXX ~ % tabcmd login -tn XXXX-ala -to XXXXXXXXX https://tableauq.XXXXX.com --no-certcheck  -t EMEA
INFO:  Creating new session
INFO:  =====   Server: https://tableauq.XXXX.com
INFO:  =====   Username: XXXXX\ala
INFO:  Tableau Server Site: XXXX
INFO:  Connecting to the server...
INFO:  Succeeded
(base) ala@XXXXXXXXX ~ % 
(base) ala@XXXXXXXXX ~ % tabcmd get "/views/XXXXXXX_16565038375510/4-XXXXX.csv"
INFO:  Read the Tableau Server password from FILE
INFO:  Creating new session
INFO:  =====   Server: https://tableauq.XXXX.com
INFO:  =====   Username: XXXXX\ala
INFO:  Tableau Server Site: XXXX
INFO:  Connecting to the server...
INFO:  Succeeded
INFO:  Unexpected response from the server:

**Comment by jacalata** [comment]
Hi, thanks for your patience! There is a bug in the current release that breaks GET. If you are able to use the development branch, it is fixed in there, and if not then there should be a new release next week.

**Comment by jacalata** [comment]
This is fixed in the latest release (2.0.7) - I'll close this now, but feel free to re-open it if there's anything missing or further work needed.
